doc/help/pypeit_ql_multislit.rst

```diff
@@ -64,7 +64,7 @@
                             (negative) B image to the left. (default: None)
       --redux_path REDUX_PATH
                             Location where reduction outputs should be stored.
-                            (default: /Users/tim/MMT/pypeit/doc)
+                            (default: /Users/westfall/Work/packages/pypeit/doc)
```

Comment:
should probably hard-code this to something generic rather than inherit from whatever env the build is done in.

Reply:
I agree that this is annoying and I put in a fix.